lib/jormungandr/src/Cardano/Wallet/Jormungandr/Network.hs

```diff
@@ -140,6 +151,9 @@ data JormungandrLayer m = JormungandrLayer
         :: Hash "BlockHeader"
         -> Word
         -> ExceptT ErrGetDescendants m [Hash "BlockHeader"]
+    , postMessage
```

Comment:
- Avoids collision with `NetworkLayer.postTx`
- Coherent with the terminology of the API
- Signals that we could submit other messages